doc/tags/for.rst

```diff
@@ -66,14 +66,52 @@ Variable              Description
 ``loop.last``         True if last iteration
 ``loop.length``       The number of items in the sequence
 ``loop.parent``       The parent context
-===================== =============================================================
+``loop.cycle``        Cycle over a sequence of values
+``loop.changed``      True if previously called with a different value or if not called yet
+``loop.previous``     The value from the previous iteration (``null`` for the first iteration)
+``loop.next``         The value from the next iteration (``null`` for the first iteration)
+===================== ========================================================================
 
 .. code-block:: twig
 
     {% for user in users %}
         {{ loop.index }} - {{ user.username }}
     {% endfor %}
 
+Use ``loop.cycle`` to cycle among a list of values:
```

Comment:
Do we need to add this one give we already implemented the standalone cycle filter (that was added later in Jinja) ?